qa/rpc-tests/hardforkdetection.py

```diff
@@ -47,10 +42,6 @@ def run_test(self):
         # Check that safe mode is on.
         self.assert_safemode_on("We do not appear to fully agree with our peers!")
 
-        # Check that an -alertnotify was triggered.
-        with open(self.alert_filename, 'r', encoding='utf8') as f:
-            alert_text = f.read()
-
```

Comment:
We still have the logic that generates this, in `CheckForkWarningConditions`. Undo the changes to this test.